bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4666,5 +4666,251 @@
+            </sqlCheck>
+        </preConditions>
+        <comment>Update version for the diagnosis concept for 'Visit Diagnoses' concept </comment>
+        <sql>
+            SET @concept_id = 0;
+
+            SELECT rmap.concept_id into @concept_id from concept_reference_map rmap, concept_name cn where cn.concept_id = rmap.concept_id and cn.name='Visit Diagnoses'
+            and cn.concept_name_type = 'FULLY_SPECIFIED' and cn.voided = 0 group by rmap.concept_id having count(*) = 1;
+
+            UPDATE concept set version = 'BAHMNI-CORE-1.0' where concept_id = @concept_id;
+        </sql>
+    </changeSet>
+
+    <changeSet id="bahmni-emrapi-202208101240" author="BAHMNI" >
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from concept c
+                INNER join
+                concept_name cn
+                on cn.concept_id=c.concept_id and cn.name='Coded Diagnosis' and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 and c.version is null;
+            </sqlCheck>
+            <sqlCheck expectedResult="1">
+                select count(*) from concept_reference_map rmap, concept_name cn where cn.concept_id = rmap.concept_id and cn.name='Coded Diagnosis'
+                and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 group by rmap.concept_id having count(*) = 1;
+            </sqlCheck>
+        </preConditions>
+        <comment>Update version for the diagnosis concept for 'Coded Diagnosis' concept</comment>
+        <sql>
+            SET @concept_id = 0;
+
+            SELECT rmap.concept_id into @concept_id from concept_reference_map rmap, concept_name cn where cn.concept_id = rmap.concept_id and cn.name='Coded Diagnosis'
+            and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 group by rmap.concept_id having count(*) = 1;
+
+            UPDATE concept set version='BAHMNI-CORE-1.0' where concept_id=@concept_id;
+        </sql>
+    </changeSet>
+
+    <changeSet id="bahmni-emrapi-202208101241" author="BAHMNI">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from concept c
+                INNER join
+                concept_name cn
+                on cn.concept_id=c.concept_id and cn.name='Non-coded Diagnosis' and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 and c.version is null;
+            </sqlCheck>
+            <sqlCheck expectedResult="1">
+                select count(*) from concept_reference_map rmap, concept_name cn where cn.concept_id = rmap.concept_id and cn.name='Non-coded Diagnosis'
+                and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 group by rmap.concept_id having count(*) = 1;
+            </sqlCheck>
+        </preConditions>
+        <comment>Update version for the diagnosis concept for 'Non-coded Diagnosis' concept</comment>
```

Comment:
Can u mention the reason for doing this in the comment? Ex. To ensure Bahmni Diagnosis valuesets are unique in DB, and don't get duplicated by OpenMRS/CIEL dictionary which also uses similar hierarchy.

Reply:
@gsluthra handled feedback comments